src/lib/merkle_mask/maskable_merkle_tree.ml

```diff
@@ -117,4 +117,24 @@ struct
     in
     ignore (unregister_mask_exn parent t_as_mask) ;
     List.iter dangling_masks ~f:(fun m -> ignore (register_mask parent m))
+
+  let batch_notify_children t accounts =
```

Comment:
I've notice people, including me, sometimes confuse which is parent and which is child.

Can we name this `batch_notify_mask_children` ?